### ISSUE TITLE
Abfrage via Koordinaten

### ISSUE THREAD

**Issue by mst94** [bundesAPI/nina-api#15]
Hallo,

ich bräuchte für meinen Standort alle NINA-Warnungen. Ich habe es so verstanden, dass ich mir erst die Liste aller Warnungen holen muss, dann für jede einzelne Warnung das GeoJSON holen muss und dort prüfen, ob mein Standort darin liegt.
Je nach Anzahl an Warnungen ist dies natürlich nicht gerade unaufwändig. Gibt es einen alternativen, einfacheren Weg?

**Comment by jumoog** [comment]
Wenn sich dein Standort nicht ändert kannst du einfach deinen AGS nehmen und so die Suche stark einschränken.

**Comment by mst94** [comment]
Ich dachte dies funktioniert nur bei den Corona-Warnungen. Ich würde dies gerne auch für Bevölkerungswarnungen / Katwarn nutzen.

**Comment by t-huyeng** [comment]
@mst94 über die folgende Abfrage sollte eine Abfrage nach AGS funktionieren: 
`https://warnung.bund.de/bbk.status/status_<AGS>.json`
Beispiel:
https://warnung.bund.de/bbk.status/status_091620000000.json

Dies ist aktuell noch nicht dokumentiert.

**Comment by buEtz** [comment]
@t-huyeng: Die "nicht API"-Schnittstellen scheinen eher noch historisch zu sein, zumindest scheinen diese mit erscheinen von Version 3 der App durch die neue Basis-URL abgelöst zu sein.

Aber auch hier gibt es offenbar (analog zum COVID-Dashboard) eine Übersicht der Warnungen auf Kreisebene, wie üblich mit den Nullen ab der sechsten Stelle mit Nullen aufgefüllt:
https://warnung.bund.de/api31/dashboard/<ARS>.json

Beispiel: https://warnung.bund.de/api31/dashboard/091620000000.json

Aber auch das müsste noch dokumentiert werden.

**Comment by t-huyeng** [comment]
Stimmt, das würde auch erklären warum bei den "alten" URLs ein paar nicht mehr funktionieren.
